pkg/porter/testdata/show/expected-output.yaml

```diff
@@ -20,3 +21,13 @@ status:
   bundleReference: getporter/wordpress:v0.1.0
   bundleVersion: 0.1.0
   bundleDigest: sha256:88d68ef0bdb9cedc6da3a8e341a33e5d2f8bb19d0cf7ec3f1060d3f9eb73cae9
+_calculated:
+  resolvedparameters:
+    - name: logLevel
+      type: integer
+      sensitive: false
```

Comment:
When a resolved parameter is sensitive, is value omitted?  Or perhaps populated with `*******`?

Do we want/need to add a param of this type for these tests?  Or covered sufficiently elsewhere?

Reply:
in plain text output we *** out the value, in json/yaml we don't so that people can actually read and use it.